website/src/pages/account/edit.tsx

```diff
@@ -0,0 +1,64 @@
+import Head from "next/head";
+import Link from "next/link";
+import Router from "next/router";
+import React, { useState } from "react";
+import { getSession, useSession } from "next-auth/react";
+import { PrismaClient, User } from "@prisma/client";
+import { Button, Input, InputGroup, Stack } from "@chakra-ui/react";
+import { Footer } from "../../components/Footer";
+import { Header } from "../../components/Header";
+
+import styles from "../styles/Home.module.css";
+
+export default function Account() {
+  const { data: session } = useSession();
+  const [username, setUsername] = useState("");
+  const updateUser = async (e: React.SyntheticEvent) => {
+    e.preventDefault();
+    try {
+      const body = { username };
+      await fetch("../api/username", {
```

Comment:
Just to ask, why is the url `../api/username`? Shouldn't `/api/username` work?

Reply:
Yeah it does, didn't catch that thanks!